python/paddle/fluid/framework.py

```diff
@@ -659,8 +659,7 @@ def fill_constant(shape, value, force_cpu=False, out=None):
                 'dtype': out.dtype,
                 'value': float(value),
                 'force_cpu': force_cpu
-            },
-            stop_gradient=True)
```

Comment:
Please make sure that it's ok to remove`stop_gradient=True` . LGTM for the rest.

Reply:
As we discussed offline,  it's ok to remove `stop_gradient=True`.